src/internal/operations/backup.go

```diff
@@ -56,6 +54,9 @@ type BackupOperation struct {
 
 	// when true, this allows for incremental backups instead of full data pulls
 	incremental bool
+	// when false it indicates we can't run delta queries on the mailbox because
+	// mailbox is full which is indicated by QuotaExceeded.
+	canMakeDeltaQueries bool
```

Comment:
BackupOperation isn't the appropriate owner for this value.  Delta queries are a graphAPI construct.  If we introduce a different backup producer, then this flag doesn't translate.  If you're looking to cache the results of `IsBackupRunnable`, then they should get cached within the backup producer (ie: GC), not in the backupOperation.